src/model.ts

```diff
@@ -281,7 +281,10 @@ export class Model extends EventBus<any> implements CommandDispatcher {
 
     this.joinSession();
 
-    if (config.snapshotRequested) {
+    if (
+      config.snapshotRequested ||
+      (data["[Content_Types].xml"] && this.config.mode !== "readonly")
```

Comment:
mode dashboard is also considered readonly
```suggestion
      (data["[Content_Types].xml"] && !this.this.getters.isReadonly())
```